src/sc/embedder/content/templates/embeddervideojs.pt

```diff
@@ -30,6 +30,7 @@
     <tal:notajax tal:condition="python:not ajax_load or ajax_include_head">
         <div tal:replace="structure provider:plone.htmlhead" />
         <link tal:replace="structure provider:plone.htmlhead.links" />
+        <link href="//vjs.zencdn.net/5.8/video-js.min.css" rel="stylesheet">
```

Comment:
@hvelarde @rodfersou There'll be interesting to implement here?

https://blog.cloudflare.com/an-introduction-to-javascript-based-ddos/
https://hacks.mozilla.org/2015/09/subresource-integrity-in-firefox-43/